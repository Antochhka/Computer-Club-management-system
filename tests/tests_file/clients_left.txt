2
09:00 12:00
10
09:00 1 clientb
09:01 1 clienta
09:02 2 clientb 1
12:00 4 clientb