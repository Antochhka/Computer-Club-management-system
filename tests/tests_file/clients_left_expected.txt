09:00
09:00 1 clientb
09:01 1 clienta
09:02 2 clientb 1
12:00 4 clientb
12:00 11 clienta
12:00
1 30 02:58
2 0 00:00